### ISSUE TITLE
[Calendar] Cursor style is not pointer when `isInteractive` prop is `true`

### ISSUE THREAD

**Issue by IAmVisco** [plouc/nivo#1861]
Should be self-explanatory from the title. For UX (and maybe a11y?) reasons I'd like to think that if `isInteractive: true`, then cursor should change style to pointer on hover.

Easily reproducible in the docs sample https://nivo.rocks/calendar/

======
Bonus points if there's condition check? Kinda going overboard here, but right now I have a situation where **some** days can have an URL, so I'd like to be able to click on the day with URL to open that URL (doable with `onClick`), and to have correct cursor style for that (not sure about that part).

**Comment by AbhaySBhosale** [comment]
Looking for the same, When user hovers over pieChart it should be hand cursor OR there should be an prop to handle styles/cursor

**Comment by plouc** [comment]
I guess a prop could be used to configure this, I used to change the cursor on some charts but some users complained about it, so I guess it depends, even if you look at GitHub's similar chart, it's not using a pointer cursor.

Not something I will work on in a near future though, but if I have to work on the Calendar at some point, I might consider it. For now, if you're using the SVG version, you can do this via CSS with some descendant selectors `svg > g > rect` iirc.

**Comment by AbhaySBhosale** [comment]
@plouc I do need it for Responsive Pie Chart, In fact it should be available for all charts to send custom styles, or arcs specific styles.

**Comment by bdefore** [comment]
Clarifying pluoc's suggestion, I was able to get a pointer cursor on a ResponsiveRadialBar chart with:

```
svg > g > g > path {
  cursor: pointer;
}
```

I did have to place this declaration on the container of the chart rather than applying styles to ResponsiveRadialBar component itself.
